xsec_2018_190702.yml

```diff
@@ -72,13 +70,13 @@
   legend: 'TTHut'
   order: 4
 
-'hist_SingleMuonRun2017.root':
+'hist_SingleMuonRun2018.root':
```

Comment:
I agree it is better for consistency, though these lines are not used for data, could simply discard them if it makes nothing crash

Reply:
This should not affect the code. It can be safely removed